aiops/opni-opensearch-update-service/opensearch-update-service/app/main.py

```diff
@@ -122,6 +167,19 @@ async def update_logs(es, df):
                     logging.info("Elasticsearch connection error")
                     es = await setup_es_connection()
 
+async def create_templates_index():
+    es = await setup_es_connection()
+    try:
+        exists = await es.indices.exists("templates")
+        if not exists:
+            await es.indices.create(index="templates")
+
+    except TransportError as exception:
+        logging.info(f"Error in es indices {exception}")
+        if exception.status_code == "N/A":
+            logging.info("Elasticsearch connection error")
+
+
```

Comment:
This should be managed as part of the operator code we have for creating indices.  We should include an index template if the document in the index will be a standard format that we control.

As an example here is where we create some index templates - https://github.com/rancher/opni/blob/main/pkg/resources/opnicluster/elastic/indices/indices.go#L193-L219

And here is where we create an index:
https://github.com/rancher/opni/blob/main/pkg/resources/opnicluster/elastic/indices/indices.go#L262-L266